### PR TITLE
Updated the description of tool "projective_transformation_points"

### PR DESCRIPTION
**Pull request by qgao-hd**
... and added new test data

### CODE REVIEW

File: tools/projective_transformation_points/projective_transformation_points.py [hunk 1]
```diff
@@ -1,8 +1,17 @@
-from skimage.transform import ProjectiveTransform
-from scipy.ndimage import map_coordinates
+"""
+Copyright 2019-2022 Biomedical Computer Vision Group, Heidelberg University.
+
+Distributed under the MIT license.
```
Comment (bgruening) [inline tools/projective_transformation_points/projective_transformation_points.py:4]: you can add to the <tool> a licence field to specify that.

File: tools/projective_transformation_points/projective_transformation_points.xml [hunk 2]
```diff
@@ -1,38 +1,45 @@
-<tool id="ip_projective_transformation_points" name="Projective Transformation" version="0.1.0">
-    <description>of input points</description>
+<tool id="ip_projective_transformation_points" name="Projective Transformation" version="0.1.1" profile="20.05">
```
Comment (bgruening) [inline tools/projective_transformation_points/projective_transformation_points.xml:1]: you can add here a `license=`

File: tools/projective_transformation_points/projective_transformation_points.xml [hunk 3]
```diff
@@ -1,38 +1,45 @@
...
+       <data format="tabular" name="output" />
     </outputs> 
     <tests>
       <test>
-        <param name="moving_points" value="points_tsv.tsv" />
-        <param name="warp_matrix" value="warp_matrix.tsv" />
-        <output name="out" value="out.tsv" ftype="tabular" />
+        <param name="input" value="in.tabular" />
+        <param name="tmat" value="tmat.tabular" />
+        <output name="output" value="out.tabular" ftype="tabular" compare="diff" />
```
Comment (bgruening) [inline tools/projective_transformation_points/projective_transformation_points.xml:29]: please use asserts here, diff on a tabular is to unspecific

Comment (qgao-hd) [reply tools/projective_transformation_points/projective_transformation_points.xml:29]: I checked the tags in assert_contents, but found none of them really appropriate for our case.

compare="diff" seems strict enough to make sure everything the same. This is not correct?

Comment (bgruening) [reply tools/projective_transformation_points/projective_transformation_points.xml:29]: oh, yes, sorry. I thought it is sim_size ... all good!

**Pull request merged**